test/unit/gateways/credorax_test.rb

```diff
@@ -553,15 +556,6 @@ def test_credit_adds_submerchant_id
     end.respond_with(successful_credit_response)
   end
 
-  def test_supports_billing_descriptor
-    @options[:billing_descriptor] = 'abcdefghijkl'
-    stub_comms do
-      @gateway.purchase(@amount, @credit_card, @options)
-    end.check_request do |_endpoint, data, _headers|
-      assert_match(/i2=abcdefghijkl/, data)
-    end.respond_with(successful_purchase_response)
-  end
-
```

Comment:
I removed this test because it is line-for-line identical to the next test in the file. They just have different names.